uncrustify.cfg

```diff
@@ -1,15 +1,17 @@
 set FOR foreach
 set WORD slots
 
-newlines			= LF
+newlines				= LF
```

Comment:
This one and some other changes confuse me...

Reply:
That is because this file is formatted with tabs.

Reply:
@evsh:  

>this file is formatted with tabs. 

Could you please make the alignment of all your `= X` fields consistent, though?  

Right now they seem to be all over the place…

Reply:
It is already. Set tab width to 4 and voilà, the values column will be aligned correctly.

Reply:
@evsh:  

>Set tab width to 4 

And how would you recommend I do that using the web-browser version of Github?

Reply:
https://github.com/evsh/qBittorrent/blob/uncrustify/uncrustify.cfg

Reply:
@evsh:  No dice.  When I click that link, the spacing is just as messy as it is displayed in the diff.  

(Note that I am using Safari on my iPod touch right now, because I cannot be bothered to get up this morning.  Nonetheless, both mobile and desktop modes for the website show the same spacing—so it might be just how iOS displays tabs…)

Reply:
That is to ensure that the formatting is not dependent on the settings of one or another editor, we use 4 spaces for indentation in our source code. I think to do here so it is a good idea.

Reply:
@evsh:  It also might be that I am blocking web fonts (among other things) with a whole slew of adblockers…

Reply:
@glassez:  

>we use 4 spaces for indentation in our source code.  

Unfortunately, @evsh decided to use actual **tab-characters** that he *renders* as 4 spaces, rather than just the spaces…

Reply:
@glassez: maybe in another PR?

@LordNyriox: I don't know. I think the only needed thing for this to work properly is a monospaced font. How one can block browser from using monospaced fonts?

Reply:
> Unfortunately, @evsh decided 

No, I did not create this file. It was formatted with tabs long before me.

Reply:
@LordNyriox if you click "view" button and then "raw" button and open in your text viewer/editor, isn't everything aligned?

Reply:
>@LordNyriox if you click "view" button and then "raw" button and open in your text viewer/editor, isn't everything aligned?

Scratch that. It is mismatched for me too.

Reply:
So, does all this matter for merging this PR? I kind of lost. Do you want me to reformat the whole file with spaces or not?